scripts/parse_new_build_warnings.py

```diff
@@ -165,11 +165,16 @@ def parse_new_build_warnings_from_directory(
 
 def export_build_warnings(warnings_dict: Dict[str, Set[str]], output: str):
     with open(output, "w") as f:
+        f.write(
+            f"# New build warnings\nA List of all additional build warnings present at this hash\n"
+        )
         for target, warnings in warnings_dict.items():
-            f.write(f"New build warnings from target: {target}\n")
+            if not len(warnings):
+                continue
+            f.write(f"## {target}\n```")
```

Comment:
missing `\n` after the \`\`\`
If each `warning` begins with a `\n`, can disregard

Reply:
technically no \n after ``` is still considered part of the code block. If it helps with readability, I can add \n after ```

Reply:
I don't think it's necessary then. I just put something in an online markdown renderer and it only rendered the part after the `\n`
that is
\`\`\`something here
asdfasdf
\`\`\`
became
```
asdfasdf
```

Since it appears we don't have the same problem, it should be fine

Reply:
Ah yeah thanks for the check.